pkg/securityhubcollector/security_hub_collector.go

```diff
@@ -19,10 +21,11 @@ import (
 
 // HubCollector is a generic struct used to hold setting info
 type HubCollector struct {
-	Logger    *zap.Logger
-	HubClient securityhubiface.SecurityHubAPI
-	Outfile   string
-	AcctMap   map[string]string
+	HubClient  securityhubiface.SecurityHubAPI
+	AcctMap    map[string]string
```

Comment:
Something feels a little weird about this interface, where you have to pass in the teamMap file name to `Initialize`, and then it parses it for you, and then you have to iterate through the parsed results and call `GetAndWriteFindingsToOutput` on each iteration. It kind of feels like, if you are asking it how to call it, then why doesn't it just do all the work itself? But I realize it's a little more complicated than that, so if you want to just leave it I'm fine with it.